packages/core/admin/admin/src/content-manager/components/Wysiwyg/EditorWrapper.js

```diff
@@ -191,7 +179,7 @@ const EditorWrapper = styled.div`
   .CodeMirror-scrollbar-filler,
   .CodeMirror-gutter-filler {
     position: absolute;
-    /* z-index: 6; */
+    z-index: 1;
```

Comment:
Does this work with our design system components z indices ?

Reply:
Yes it does, this is why I have set it to 1.